app/models/site.js

```diff
@@ -23,11 +23,14 @@ export default class SiteModel extends AbstractValidationModel {
   })
   siteType;
 
+  @attr siteTypeName;
+
   get validationSchema() {
     return Joi.object({
       address: validateBelongsToOptional(),
       contacts: validateHasManyOptional(),
       siteType: validateBelongsToRequired('Voeg een type vestiging toe'),
+      siteTypeName: Joi.string().allow('').optional(),
```

Comment:
**Nitpick:**  As I understand it specifying both `allow('')` and `optional()` is redundant here. Specifying `allow('')` (or `empty('') as done in [another model](https://github.com/lblod/frontend-organization-portal/blob/2698fe92c0a3e9584a146467e2defd6cc00f0145/app/models/organization.js#L101)) should be sufficient.

Reply:
yeah, had a small fight with Joi for this PR xd, and this is what it worked, but you are right, empty("") works also fine